tests/Knp/Menu/Tests/Provider/ArrayAccessProviderTest.php

```diff
@@ -18,6 +18,7 @@ public function testHas(): void
 
     public function testGetExistentMenu(): void
     {
+        /** @var \ArrayObject<string, mixed> $registry */
```

Comment:
should be `ArrayObject<string, ItemInterface|callable>` to match the type of the ArrayAccessProvider